se/Mortgage/v0/schema/definitions/Applicant.yaml

```diff
@@ -0,0 +1,224 @@
+"$id": "https://open-broker.org/se/Mortgage/v0/schema/definitions/Applicant"
+"$schema": "http://json-schema.org/draft-07/schema#"
+
+title: Applicant
+description: |
+  Describe the attributes of an applicant, such as housing situation,
+  employment details, financial information, marital status and citizenship.
+
+type: object
+additionalProperties: false
+required:
+  - ssn
+  - employmentStatus
+  - employmentStatusSinceYear
+  - employmentStatusSinceMonth
+  - dependentChildren
+  - paymentRemarks
+  - housingType
+  - monthlyRent
+  - monthlyGrossIncome
+  - monthlyNetIncome
+  - maritalStatus
+
+properties:
+  ssn:
+    type: string
+    title: '12-digit Social Security Number (sv. Personnr)'
+    description: "A Swedish Social-security number on the 12-digit format YYYYMMDDXXXX"
+    pattern: "^[0-9]{12}$"
+    minLength: 12
+    maxLength: 12
+    examples:
+      - '195911057916'
+  phone:
+    type: string
+    title: "Phone number to the applicant"
+    pattern: '^\+[1-9][0-9]{1,14}$'
```

Comment:
Consider: Always use double quotes (or single quotes) for pattern values consistently. Especially since this seems to have different effects on the regex itself.

Reply:
Yeah, I agree. Maybe this is better with single quotes, because then you don't have to escape backslashes all the time.

Reply:
Fixed